README.md

```diff
@@ -55,7 +55,7 @@ compile 'com.vanniktech:lint-rules-rxjava2:0.10.0-SNAPSHOT
 - **RxJava2DefaultScheduler** - Calling this method will rely on a default scheduler. This is not necessary the best default. Being explicit and taking the overload for passing one is preferred.
 - **RxJava2DisposableAddAllCall** - Instead of using addAll(), add() should be used separately for each Disposable.
 - **RxJava2DisposableDisposeCall** - Instead of using dispose(), clear() should be used. Calling clear will result in a CompositeDisposable that can be used further to add more Disposables. When using dispose() this is not the case.
-- **RxJava2MethodMissingCheckReturnValue** - Methods returning RxJava Reactive Types should be annotated with the @CheckReturnValue annotation. Static analyze tools such as Lint or ErrorProne can detect when the return value of a method is not used. This is usually an indication of a bug. If this is done on purpose (e.g. fire & forget) it should be stated explicitly.
+- **RxJava2MethodMissingCheckReturnValue** - Methods returning RxJava Reactive Types should be annotated with the @CheckReturnValue annotation. Static analyze tools such as Lint or ErrorProne can detect when the return value of a method is not used. This is usually an indication of a bug. If this is done on purpose (e.g. fire & forget) it should be stated explicitly. **Note:** Methods with certain visibility modifiers (e.g private) can be ignored by providing a System property named: `com.vanniktech.lintrules.rxjava2.RxJava2MethodMissingCheckReturnValueDetector.ignoreMethodAccessModifiers`. 
```

Comment:
Can you revert this change? I'll update this once I've published a new version.

Reply:
I did this in a separate commit anyways. Just removing the commit now.